api/src/main/java/com/example/api/service/activity/task/GraphTaskService.java

```diff
@@ -127,16 +125,30 @@ public void createGraphTask(CreateGraphTaskChapterForm chapterForm) throws Parse
         }
         questionRepo.saveAll(questions);
 
+        double maxPoints = calculateMaxPoints(questions.get(0), 0);
```

Comment:
Mogę się mylić, ale wydaje mi się, że powinniśmy wziąść max z calculateMaxPoints(questions.get(i), questions.get(i).getPoints()) gdzie i to indeks pytania z pierwszej puli (chyba, że pierwsze pytanie jest jedno, ale wtedy i tak chyba trzeba podać jako drugi argument liczbę punktów za to pytanie

Reply:
Pierwse pytanie jest jedno i to jest pytanie tak zwane 'puste', nie ma za nie punktów. Chodzi o to, że front z tego pytania z pola nextQuestions pobiera sobie te prawdziwe pierwsze pytania i je wyświetla, dlatego tam jest get(0) i 0 pkt bo to pierwsze pytanie zawsze jest jedno i jest warte nic.

Reply:
Jak tak to git